locale/es/download/current.md

```diff
@@ -12,7 +12,7 @@ downloads:
     intro: >
         Descargue el código fuente de Node.js o un instalador pre-compilado para su plataforma, y comience a desarrollar hoy.
     currentVersion: Versión actual
-    buildInstructions: Building Node.js from source on supported platforms
+    buildInstructions: Compilando Node.js desde el código fuente en las plataformas soportadas
```

Comment:
I don't know Spanish,
But Google translator (which is not that accurate) is translating this Spanish into English as:
> Compiling Node.js from source code on supported platforms.

And when I do original English -> Spanish
> Construyendo Node.js desde la fuente en las plataformas compatibles.

Reply:
👍 for fixing this. :)

Reply:
I added the translation suggested in #1608. I can make any change if needed :+1:

Reply:
I think it's fine as it is, native speakers already agreed about this translation in #1608